### DISCUSSION TITLE
Is there a way to add attributes with capital letters? i.e. '[ngStyle]'

### DISCUSSION THREAD

**Discussion by nathanbrinqa** [apostrophecms/sanitize-html#500] [Q&A]
I am using angular and would like to add [ngStyle] to the list of allowedAttribtues.

**Comment by abea** [comment]
sanitize-html uses htmlparser2 and can use all its options in the `parser` object. I see htmlparser2 has a [`lowerCaseAttributeNames`](https://github.com/fb55/htmlparser2/wiki/Parser-options#option-lowercaseattributenames) you could try setting to `false` for this. The docs say it defaults to false, but an [old issue discussion](https://github.com/fb55/htmlparser2/issues/212) suggests that might not be the case.
